src/main/java/org/folio/service/loader/MarcLoadResult.java

```diff
@@ -0,0 +1,31 @@
+package org.folio.service.loader;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MarcLoadResult {
```

Comment:
shouldn't this be a schema file? and be generated rather than create a POJO ourselves?